src/core/egg-herbie.rkt

```diff
@@ -753,21 +713,13 @@
 
   ; convert id->spec to a vector-map
   (define specs (make-vector n #f))
-  (for ([(id spec&repr) (in-dict id->spec)])
+  (for ([(id spec&repr) (in-hash id->spec)])
```

Comment:
Performance?

Reply:
Not really. I was just inlining some code.